### PR TITLE
digikam: fix media playback by adding ffmpeg to buildInputs

### PR DESCRIPTION
**Pull request by callahad**
###### Motivation for this change

Since Digikam 6.0.0, media playback requires `ffmpeg` as a direct dependency.

This shouldn't really change the closure size, as another of Digikam's dependencies, `libqtav`, already depends on `ffmpeg`.

The difference is evident during the `cmakeConfigurePhase`:

```patch
--- before.txt	2019-02-21 10:10:24.312731211 +0000
+++ after.txt	2019-02-21 10:01:43.642602245 +0000
@@ -1,13 +1,13 @@
--- Could NOT find FFmpeg (missing: FFMPEG_LIBRARIES FFMPEG_INCLUDE_DIRS AVCODEC_LIBRARIES AVCODEC_INCLUDE_DIRS AVFILTER_LIBRARIES AVFILTER_INCLUDE_DIRS AVFORMAT_LIBRARIES AVFORMAT_INCLUDE_DIRS AVUTIL_LIBRARIES AVUTIL_INCLUDE_DIRS SWSCALE_LIBRARIES SWSCALE_INCLUDE_DIRS) 
--- FFMPEG_FOUND        = FALSE
--- FFMPEG_INCLUDE_DIRS = 
--- FFMPEG_LIBRARIES    = 
--- FFMPEG_DEFINITIONS  = 
+-- Found FFmpeg: /nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libavcodec.so;/nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libavfilter.so;/nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libavformat.so;/nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libavutil.so;/nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libswscale.so
+-- FFMPEG_FOUND        = TRUE
+-- FFMPEG_INCLUDE_DIRS = /nix/store/bsgkvbqniycnf94xv8dmlszazqrlas7n-ffmpeg-3.4.5-dev/include
+-- FFMPEG_LIBRARIES    = /nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libavcodec.so;/nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libavfilter.so;/nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libavformat.so;/nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libavutil.so;/nix/store/naf33s4w5kpxv060vmjv6cz5jvcqnxrb-ffmpeg-3.4.5/lib/libswscale.so
+-- FFMPEG_DEFINITIONS  =
 -- QtAV search path: /nix/store/yjj7f7lypknwzbxvkvjkf8aa0av6ss0d-qtbase-5.12.0-dev/lib/cmake
 -- Found QtAV: /nix/store/ncxf8s0jlqmh1cvlp9day3byh45mxqb3-libqtav-1.12.0/lib/libQtAV.so;/nix/store/ncxf8s0jlqmh1cvlp9day3byh45mxqb3-libqtav-1.12.0/lib/libQtAVWidgets.so
 -- Found QtAV version 1.12.0
 -- QtAV_FOUND       = TRUE
 -- QtAV_INCLUDE_DIR = /nix/store/ncxf8s0jlqmh1cvlp9day3byh45mxqb3-libqtav-1.12.0/include/QtAV /nix/store/ncxf8s0jlqmh1cvlp9day3byh45mxqb3-libqtav-1.12.0/include/QtAVWidgets
 -- QtAV_LIBRARIES   = /nix/store/ncxf8s0jlqmh1cvlp9day3byh45mxqb3-libqtav-1.12.0/lib/libQtAV.so;/nix/store/ncxf8s0jlqmh1cvlp9day3byh45mxqb3-libqtav-1.12.0/lib/libQtAVWidgets.so
 -- QtAV_VERSION     = 1.12.0
--- ENABLE_MEDIAPLAYER option is enabled but FFMpeg cannot be found. Media player support is disabled.
+-- Media player support is enabled.
```

...and during program execution, in the Help → Components Information menu:

digikam-ffmpeg

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aanderse** [comment]
@GrahamcOfBorg build digikam
@callahad next time no need for an issue if you already have a PR.

**Comment by callahad** [comment]
Gotcha. For next time, what's the best way to get a pull request reviewed after it's been sitting for a few days?

**Comment by aanderse** [comment]
Timed out... I'll try this again then:

@GrahamcOfBorg build digikam

@callahad to get your PRs merged you want to make them more appealing to people with commit access. You should look for people who might not have commit access, but have been doing a fair bit of work and review on nixpkgs. If you get people like this to review and approve your PR then people with commit access will be more like to accept your PR as the PR has already been reviewed and the amount of work required has been lowered.

https://github.com/NixOS/nixpkgs/issues/50105#issuecomment-452435126

Another effective way I've found has simply been to bug people on IRC or discourse until they merge your PRs :smiley:

**Comment by aanderse** [comment]
@GrahamcOfBorg build enblend-enfuse vigra python27Packages.numpy openblas

**Comment by aanderse** [comment]
Builds on my system. The aarch64-linux issue seems to be with dependencies and unrelated to this PR.

**Review by aanderse — APPROVED** [review]
This builds on my system.
LGTM :+1:

**Comment by aanderse** [comment]
@markuskowa seeing as you're aware of the openblas issue on aarch64 issue, is there any chance you'd be willing to merge?

**Comment by aanderse** [comment]
@markuskowa Thanks! :tada:

**Comment by markuskowa** [comment]
Thanks. LGTM. Let hydra do the build. This seems to be too big for OfBorg.

**Pull request merged**
